src/languages/en.ts

```diff
@@ -1021,6 +1021,7 @@ const translations = {
         bookingPendingDescription: "This booking is pending because it hasn't been paid yet.",
         bookingArchived: 'This booking is archived',
         bookingArchivedDescription: 'This booking is archived because the trip date has passed. Add an expense for the final amount if needed.',
+        paymentComplete: 'Payment Complete',
```

Comment:
```suggestion
        paymentComplete: 'Payment complete',
```

i think it should be like this

Reply:
@flodnv wdyt?

Reply:
I agree, like everything else around it. Same in Spanish.